index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
+	to make sure specifications are robust in terms of accessibility, internationalisation,
+	privacy, security, and technical architecture by the time they become a [=W3C Recommendation=]. 
+
+	Horizontal review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, so that accessibility, internationalisation, privacy,
+	security, and technical architecture are considered throughout. 
+
+	Before a proposed specification transitions onto the Recommendation Track, the
+	group responsible for the specification <em class="rfc2119">must</em> solicit a
+	horizontal review from each of the horizontal review groups (in many cases,
+	submitting a self-review to the reviewing group will satisfy this requirement).
+
+	The <dfn id=dfn-horizontal-review-groups export>Horizontal Review Groups</dfn> are:
+
+	<ul>
+		<li><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a></li>
+		<li><a href="https://www.w3.org/International/core/Overview">Internationalisation Working Group</a></li>
+		<li><a href="https://www.w3.org/Privacy/">Privacy Interest Group</a></li>
+		<li><a href="https://www.w3.org/2001/tag/">Technical Architecture Group</a></li>
+	</ul>
+
+	When the [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to
+	<a href="https://lists.w3.org/Archives/Public/horizontal-review-announce/">horizontal-review-announce@w3.org</a>. 
+
+	Horizontal review <em class="rfc2119">must</em> be solicited from the Horizontal
+	Review groups at least 60 days before requesting transition to [=Candidate Recommendation=]. 
+
+	Horizontal Review of all new and significantly revised sections of a specification
+	<em class="rfc2119">should</em> also be sought from the Horizontal Review groups. 
+
+	When a Working Group requests horizontal review: 
+
+	<ul>
+		<li>The horizontal review group <em class="rfc2119">must</em> acknowledge
+		 	receipt of the request and indicate whether the review will be conducted</li>
+		<li>The horizontal review group <em class="rfc2119">should</em> review
+			the specification and raise relevant issues within the agreed timeframe</li>
+		<li>The requesting Working Group <em class="rfc2119">must</em> acknowledge
+			receipt of each issue, then give each issue due consideration before
+			providing an appropriate response</li>
+		<li>If the Working Group and the horizontal review group do not agree,
+			they <em class="rfc2119">should</em> make a determined effort to find
+			agreement</li>
+		<li>When the Working Group requests transition of a specification, evidence
+			of this process <em class="rfc2119">must</em> be provided and any issues
+			that remain unresolved <em class="rfc2119">must</em> be indicated</li>
+	</ul>
+
+	The Director <em class="rfc2119">must</em> consider evidence of horizontal
+	review before approving transitions. Appropriate evidence MAY include: 
```

Comment:
This include list doesn't tally up with the must-do list above. If the must-do list has been completed then the evidence is exactly enumerable, for each HR group, as:

1. documentation that the HR group did not consider a review to be necessary OR
2. documentation that at least 60 days have elapsed since the HR request was made, in case a 'no response' is being cited OR
3. documentation that the HR group completed their review and raised no issues OR
4. documentation of both:
4.1.     the list of successfully resolved HR group's issues AND
4.2     the list of all remaining unresolved HR issues including, for each of them, evidence that a determined effort was made to find a resolution

Reply:
The *must* part of this sentence is redundant with the entry criterion of relevant stages on the REC track, which already call for this review.

Reply:
I've updated the list per @nigelmegitt's suggestion, but I'm not sure what you mean in your comment @frivoal?